src/main/java/core/basesyntax/service/DataLineServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+public class DataLineServiceImpl implements DataLineService {
+    private static final int FRUIT_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
+    @Override
+    public String getFruitName(String string) {
+        return string.split(",")[FRUIT_INDEX];
+    }
+
+    @Override
+    public int getFruitAmount(String string) {
+        return Integer.parseInt(string.split(",")[AMOUNT_INDEX]);
+    }
```

Comment:
here should be only one method that converts List of lines to List of fruits, then using for loop you go through list of fruits and call needed operation's handler